testsuite/tests/prometheus/apicast/selfmanaged/test_batching_caching_policy.py

```diff
@@ -5,11 +5,11 @@
 import pytest
 
 from packaging.version import Version  # noqa # pylint: disable=unused-import
-from testsuite import TESTED_VERSION, rawobj  # noqa # pylint: disable=unused-import
+from testsuite import TESTED_APICAST_VERSION, rawobj  # noqa # pylint: disable=unused-import
```

Comment:
APICAST_OPERATOR_VERSION sounds better. For example we could improve coverage of testing with different apicast version, in such case we could introduce also APICAST_TEMPLATE_VERSION. Better to be prepared in this case

Reply:
renamed and added template version for now it's guessed from 3scale project